pgactivity/pg.py

```diff
@@ -50,11 +50,12 @@ def load(self, data: Buffer) -> str:
             return data.decode(errors="replace")
 
     def connect(dsn: str, **kwargs: Any) -> Connection:
-        # Set client_encoding to 'auto', if unset by the user.
-        # This is (more or less) what's done by psql.
-        conninfo = conninfo_to_dict(dsn)
-        conninfo.setdefault("client_encoding", "auto")
-        dsn = make_conninfo(**conninfo)
+        if "PGCLIENTENCODING" not in os.environ:
+            # Set client_encoding to 'auto', if unset by the user.
```

Comment:
``
```suggestion
            # Set client_encoding to 'auto', if it's not set.
```

On most installation I've seen, it's not set.